### DISCUSSION TITLE
Translation validity checker for next-intl?

### DISCUSSION THREAD

**Discussion by indicozy** [amannn/next-intl#1544] [Ideas]
I want to have a script that will ensure that all my translation keys are valid. For example, if there is no such key as `t('some.non_valid.key)` I want to be sure that it won't slip to production.

I have found this project: https://github.com/gamcoh/translations-checker which solved that issue but I wanted to know is there solution baked in next-intl or any other similar solution??

**Comment by amannn** [comment]
Have you seen https://next-intl-docs.vercel.app/docs/workflows/typescript?

**Reply by indicozy** [reply]
We don't use Typescript in our company unfortunately

**Reply by amannn** [reply]
Oh, got it—that would have helped.
